template/src/todo/todo.controller.ts

```diff
@@ -32,9 +35,23 @@ export class TodoController {
   @ApiResponse({ status: HttpStatus.OK, type: TodoDTO })
   @ApiResponse({ status: HttpStatus.BAD_REQUEST, type: ApiException })
   @ApiOperation(getOperationId('Todo', 'GetAll'))
-  async getTodos(): Promise<TodoDTO[]> {
+  async getTodos(@Query() params?: any): Promise<TodoDTO[]> {
     try {
-      return plainToClass(TodoDTO, await this._todoService.findAll(), {
+      let filter: FindManyOptions<ObjectLiteral> = {};
+      let orderValue, whereValue: any;
```

Comment:
We should remove `any` types as much as we can. I prefer to use union types as `string | number` in order to consider alternatives that we already know.